pkg/io/video/convert.go

```diff
@@ -24,15 +24,18 @@ func ToI420(r Reader) Reader {
 				addrSrc := i * 2 * yuvImg.CStride
 				addrDst := i * yuvImg.CStride / 2
 				for j := 0; j < yuvImg.CStride/2; j++ {
-					cb := uint16(yuvImg.Cb[addrSrc+j]) + uint16(yuvImg.Cb[addrSrc+yuvImg.CStride+j]) +
-						uint16(yuvImg.Cb[addrSrc+j+1]) + uint16(yuvImg.Cb[addrSrc+yuvImg.CStride+j+1])
-					cr := uint16(yuvImg.Cr[addrSrc+j]) + uint16(yuvImg.Cr[addrSrc+yuvImg.CStride+j]) +
-						uint16(yuvImg.Cr[addrSrc+j+1]) + uint16(yuvImg.Cr[addrSrc+yuvImg.CStride+j+1])
+					j2 := j * 2
+					cb := uint16(yuvImg.Cb[addrSrc+j2]) + uint16(yuvImg.Cb[addrSrc+yuvImg.CStride+j2]) +
+						uint16(yuvImg.Cb[addrSrc+j2+1]) + uint16(yuvImg.Cb[addrSrc+yuvImg.CStride+j2+1])
+					cr := uint16(yuvImg.Cr[addrSrc+j2]) + uint16(yuvImg.Cr[addrSrc+yuvImg.CStride+j2]) +
+						uint16(yuvImg.Cr[addrSrc+j2+1]) + uint16(yuvImg.Cr[addrSrc+yuvImg.CStride+j2+1])
```

Comment:
Oops. I missed this one... So, we've been getting only top left Chromas and they got mixed up too haha. I'm surprised that we're still seeing the video well. I guess we're that bad with colors lol